Index: src/main/java/Sensoren/Dockerfile
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>FROM maven:3.6.3-jdk-8 AS BUILD\r\n#Kopiere alle Dateien\r\nCOPY pom.xml /tmp/\r\nCOPY src /tmp/src/\r\nWORKDIR /tmp/\r\n\r\n\r\nRUN mvn compile\r\n\r\n#Finaler Container\r\nFROM openjdk:11-jdk\r\n\r\n\r\nCOPY --from=BUILD /tmp/target /app/target\r\nWORKDIR /app\r\nEXPOSE 8080\r\nENTRYPOINT [\"java\",\"-cp\",\"/app/target/classes\",\"Sensoren.mainClass\"]
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/Sensoren/Dockerfile	(revision 5e93a01be0141a91d50a55c8fc2dc7eb7323d867)
+++ src/main/java/Sensoren/Dockerfile	(date 1591006385811)
@@ -8,7 +8,7 @@
 RUN mvn compile
 
 #Finaler Container
-FROM openjdk:11-jdk
+FROM openjdk:11
 
 
 COPY --from=BUILD /tmp/target /app/target
Index: src/main/java/ProviderServer/Dockerfile
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>FROM maven:3.6.3-jdk-8 AS BUILD\r\n#Kopiere alle Dateien\r\nCOPY pom.xml /tmp/\r\nCOPY src /tmp/src/\r\nWORKDIR /tmp/\r\n\r\n\r\nRUN mvn compile\r\n\r\n#Finaler Container\r\nFROM openjdk:11-jdk\r\nCOPY --from=BUILD /tmp/target /app/target\r\nWORKDIR /app\r\nENTRYPOINT [\"java\",\"-cp\",\"/app/target/classes\",\"ProviderServer.ProviderServer\"]
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/ProviderServer/Dockerfile	(revision 5e93a01be0141a91d50a55c8fc2dc7eb7323d867)
+++ src/main/java/ProviderServer/Dockerfile	(date 1591006385807)
@@ -8,7 +8,7 @@
 RUN mvn compile
 
 #Finaler Container
-FROM openjdk:11-jdk
+FROM openjdk:11
 COPY --from=BUILD /tmp/target /app/target
 WORKDIR /app
 ENTRYPOINT ["java","-cp","/app/target/classes","ProviderServer.ProviderServer"]
\ No newline at end of file
Index: src/main/java/Zentral/Dockerfile
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>FROM maven:3.6.3-jdk-8 AS BUILD\r\n#Kopiere alle Dateien\r\nCOPY pom.xml /tmp/\r\nCOPY src /tmp/src/\r\nWORKDIR /tmp/\r\n\r\n\r\nRUN mvn compile\r\n\r\nFROM openjdk:11-jdk\r\n\r\nCOPY --from=BUILD /tmp/target /app/target\r\nWORKDIR /app\r\nENTRYPOINT [\"java\",\"-cp\",\"/app/target/classes\",\"Zentral.Zentral\"]
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/Zentral/Dockerfile	(revision 5e93a01be0141a91d50a55c8fc2dc7eb7323d867)
+++ src/main/java/Zentral/Dockerfile	(date 1591006385814)
@@ -7,7 +7,7 @@
 
 RUN mvn compile
 
-FROM openjdk:11-jdk
+FROM openjdk:11
 
 COPY --from=BUILD /tmp/target /app/target
 WORKDIR /app
